core/templates/pages/blog-dashboard-page/blog-dashboard-tile/blog-dashboard-tile.component.ts

```diff
@@ -36,18 +38,22 @@ export class BlogDashboardTileComponent implements OnInit {
   @Input() activeView!: string;
   @Input() blogPostIsPublished: boolean = false;
   lastUpdatedDateString: string = '';
+  summaryContent!: string;
   @Output() unpublisedBlogPost: EventEmitter<void> = new EventEmitter();
   @Output() deletedBlogPost: EventEmitter<void> = new EventEmitter();
   constructor(
     private blogDashboardPageService: BlogDashboardPageService,
     private blogPostEditorBackendService: BlogPostEditorBackendApiService,
     private ngbModal: NgbModal,
-    private alertsService: AlertsService
+    private alertsService: AlertsService,
+    private truncatePipe: TruncatePipe,
   ) {}
 
   ngOnInit(): void {
     const lastUpdated = this.blogPostSummary.lastUpdated;
     this.lastUpdatedDateString = this.getDateStringInWords(lastUpdated);
+    this.summaryContent = this.truncatePipe.transform(
+      this.blogPostSummary.summary, 220);
```

Comment:
Can you please add a comment explaining what this does?

Reply:
Added.